openshift/kustomize/services/extract-quotes/overlays/dev/config-map.yaml

```diff
@@ -0,0 +1,43 @@
+---
+kind: ConfigMap
```

Comment:
not much of an overlay here.  Does this essentially replace the configs?

Reply:
yes, this whole PR is just setting a baseline.  I'm sure there will be config specific to the action of extracting the quotes that comes later